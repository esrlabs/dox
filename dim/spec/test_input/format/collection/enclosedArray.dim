module: enclosed_array

enclosed:
    -    enclosedString.dim
    - enclosedArray.dim
