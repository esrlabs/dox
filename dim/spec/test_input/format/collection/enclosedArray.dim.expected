document: enclosed_array

enclosed:
  - enclosedString.dim
  - enclosedArray.dim
